### PR TITLE
Private Route, database update

### PR DESCRIPTION
**Pull request by pritam-kr**

### CODE REVIEW

**Review by srishti-maurya — APPROVED** [review]
Looks clean and good

File: src/App.js [hunk 1]
```diff
@@ -12,6 +20,10 @@ function App() {
   const { auth } = state;
   const token = auth.token;
 
+  const PrivateRoute = ({ children }) => {
+    return token ? children : <Navigate to="/" replace />;
```
Comment (horsemaker) [inline src/App.js:24]: Maybe you could use the location's state here & navigate the user back to where he/she came from.

**Review by horsemaker — APPROVED** [review]
Just the PrivateRoute suggestion & nothing else. Everything else looks clean! 👌

File: src/App.js (hunk 1 above)
Comment (pritam-kr) [reply src/App.js:24]: Ok

**Pull request merged**